src/pages/model/model-page/model-page.js

```diff
@@ -28,6 +29,8 @@ const tableTitles = config.tableHeadRowTitles.models;
 
 const ModelPage = () => {
   const styles = useStyles();
+  const common = useCommonStyles();
```

Comment:
Can you rename this `common` variable to styles or classes?

Reply:
fixed